General/general_shacl.ttl

```diff
@@ -0,0 +1,206 @@
+        [ skos:example "ASAM OpenDRIVE / ASAM OpenSCENARIO / Road5 / Here Live Map / Lanelet / DLM / road2sim / roadXML  / Shape" ;
+            sh:datatype xsd:string ;
+            sh:description "Format type definition"@en ;
+            sh:maxCount "1"^^xsd:unsignedInt ;
+            sh:message "Validation of type failed!"@en ;
+            sh:name "type"@en ;
+            sh:order "0"^^xsd:unsignedInt ;
+            sh:path general:type ] ;
+    sh:targetClass general:Format .
+
+general:LicenceShape a sh:NodeShape ;
+    sh:property [ skos:example "link to licence" ;
+            sh:datatype xsd:anyURI ;
+            sh:description "Reference to licence"@en ;
+            sh:maxCount "1"^^xsd:unsignedInt ;
+            sh:message "Validation of link failed!"@en ;
+            sh:name "link"@en ;
+            sh:order "1"^^xsd:unsignedInt ;
+            sh:path general:link ],
+        [ skos:example "MIT" ;
+            sh:datatype xsd:string ;
+            sh:description "Name of licence type"@en ;
+            sh:maxCount "1"^^xsd:unsignedInt ;
+            sh:message "Validation of type failed!"@en ;
+            sh:name "type"@en ;
+            sh:order "0"^^xsd:unsignedInt ;
+            sh:path general:type ] ;
+    sh:targetClass general:Licence .
+
+general:LinksShape a sh:NodeShape ;
+    sh:property [ skos:example "link to CRG" ;
+            sh:description "Reference to dependent data"@en ;
+            sh:message "Validation of data failed!"@en ;
+            sh:minCount "0"^^xsd:unsignedInt ;
+            sh:name "data"@en ;
+            sh:node general:LinkShape ;
+            sh:order "1"^^xsd:unsignedInt ;
+            sh:path general:data ],
+        [ skos:example "link to screenshot, video, routing" ;
+            sh:description "Reference to screenshot data"@en ;
+            sh:message "Validation of media failed!"@en ;
+            sh:name "media"@en ;
+            sh:node general:LinkShape ;
+            sh:order "2"^^xsd:unsignedInt ;
+            sh:path general:media ],
+        [ skos:example "link to asset data, e.g. hd map" ;
+            sh:datatype xsd:anyURI ;
+            sh:description "Id / url to asset file"@en ;
+            sh:maxCount "1"^^xsd:unsignedInt ;
+            sh:message "Validation of asset failed!"@en ;
+            sh:name "asset"@en ;
+            sh:order "0"^^xsd:unsignedInt ;
+            sh:path general:asset ] ;
+    sh:targetClass general:Links .
+
+general:PositionShape a sh:NodeShape ;
+    sh:property [ skos:example "Lat: 1.234- 1.345; Lon 45.23-45.34" ;
+            sh:description "Bounding box with lat/lon values in WGS84"@en ;
+            sh:maxCount "1"^^xsd:unsignedInt ;
+            sh:message "Validation of bounding failed!"@en ;
+            sh:name "bounding"@en ;
+            sh:node general:BoundingBoxShape ;
+            sh:order "1"^^xsd:unsignedInt ;
+            sh:path general:bounding ],
+        [ skos:example "Germany, GER" ;
```

Comment:
there may already exist a standard vocabulary describing country? If so, the external node can be referenced like this:
```
 [
        sh:path general:country ;
        sh:node <http://example.com/external/country-shape> ;
        sh:maxCount "1"^^xsd:unsignedInt ;
        sh:order "0"^^xsd:unsignedInt ;
        sh:description "Country code as ISO 3166-1, alpha-2; "@en ;
        sh:message "Validation of country failed!"@en ;
        skos:example "Germany, GER" ;
    ] .
```
or as an alternative with prefix pointing to the external shape.

With this you would be able to reference to a single attribute of an external node:

```
    sh:property [
        sh:path ex:name ;
        sh:node ex:CountryShape ;
    ] .
```

Reply:
is moved to its own shape “Georeference” and added per asset data shacle
use gx:countryCode

Reply:
I cannot find gx:countryCode anywhere and the section is still unchanged:
<img width="448" alt="image" src="https://github.com/GAIA-X4PLC-AAD/ontology-management-base/assets/70939848/db964b49-fa4a-4f18-a490-e65c6a2aeb3e">

Reply:
is now part of the GeoReferenceShape contained in the special DataAssets Shacl